store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonReader.java

```diff
@@ -155,6 +154,17 @@ public static CarbonReaderBuilder builder(String tablePath) {
     return builder(tablePath, tableName);
   }
 
+  /**
+   * Return a new {@link CarbonReaderBuilder} instance
+   *
+   * @return CarbonReaderBuilder object
+   */
+  public static CarbonReaderBuilder builder() {
```

Comment:
This changes is not related to binary, please keep in separate PR

Reply:
removed